pkg/cmd/completion.go

```diff
@@ -33,6 +34,53 @@ func newCompletionCmd() *completionCmd {
 	return cc
 }
 
+var zsh_completion_instructions = `
+Suggested next steps:
+----------------------
```

Comment:
We're missing this header in the other two instruction sets.

Maybe it makes sense for the header to be printed separately from the instructions so that we don't need to include it in each one.